tools/serverpod_cli/bin/serverpod_cli.dart

```diff
@@ -239,12 +239,9 @@ Future<void> _main(List<String> args) async {
         return;
       }
 
-      var re = RegExp(r'^[a-z0-9_]+$');
-      if (results.arguments.length > 1 && re.hasMatch(name)) {
-        await performCreate(name, verbose, template, force);
-        _analytics.cleanUp();
-        return;
-      }
+      await performCreate(name, verbose, template, force);
```

Comment:
Removed the regex check cause it was added internally in the `performCreate` function